UI/MapboxNavigationUI/TurnArrowView.swift

```diff
@@ -0,0 +1,131 @@
+import UIKit
+import MapboxDirections
+import MapboxNavigation
+import SDWebImage
+
+@IBDesignable
+public class TurnArrowView: UIView {
+    var imageView: UIImageView!
+    
+    var showsShield = true
+    public var step: RouteStep? {
+        didSet {
+            imageView.isHidden = true
+            if showsShield, let components = step?.codes?.first?.components(separatedBy: " "), components.count > 1 {
+                let network = components[0]
+                let number = components[1]
+                if var imageName = ShieldImageNamesByPrefix[network] {
+                    imageName = imageName.replacingOccurrences(of: " ", with: "_").replacingOccurrences(of: "{ref}", with: number)
+                    let url = URL(string: "https://commons.wikimedia.org/w/thumb.php?f=\(imageName)&w=\(imageView.bounds.width * UIScreen.main.scale)")
```

Comment:
Yes. It will only request once per url in this case. Even after restarting the process, it will not make the request again but instead fetch it from disk. `maxage` defaults to one week.